lib/simulator-xcode-8.js

```diff
@@ -53,6 +53,29 @@ class SimulatorXcode8 extends SimulatorXcode7 {
     }
   }
 
+  async openUrl (url) {
+    const SAFARI_STARTUP_TIMEOUT = 15 * 1000;
+
+    if (!await this.isRunning()) {
+      throw new Error(`Tried to open ${url}, but Simulator is not in Booted state`);
+    }
+    const launchTimestamp = process.hrtime();
+    try {
+      await waitForCondition(async () => {
+        try {
+          await exec('bash', ['-c', `xcrun simctl launch ${this.udid} com.apple.mobilesafari ${url}`]);
```

Comment:
I think it is only necessary if you are doing things like piping from on process to another (which `exec` cannot handle on its own, so `bash` is needed). Otherwise it is forking `bash` in order for it to fork `xcrun`, which is heavier than necessary, I think.